src/components/EventCard/EventCard.tsx

```diff
@@ -35,12 +28,20 @@ export default function EventCard({
 
   return (
     <article className={styles.container}>
-      <img className={styles.img} src={imgSrc} alt="" />
-      <h2 className={styles.eventName}>{name}</h2>
+      <img
+        className={styles.img}
+        src={
+          image == null
+            ? "https://images.unsplash.com/photo-1543145480-b8d11d3581a6?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=929&q=80"
+            : image
+        }
+        alt=""
+      />
+      <h2 className={styles.eventName}>{title}</h2>
       <p>{eventDate}</p>
       <p>{time}</p>
```

Comment:
This might be a chance to use the `time` element.

Reply:
The API that I use returns a string like "It starts at 8pm", that's why I think here I can't use it.